api/handlers.go

```diff
@@ -130,22 +144,10 @@ func DeleteAllSessionsHandlerFunc(cache Cache) http.HandlerFunc {
 
 func writeErrorResponse(ctx context.Context, w http.ResponseWriter, msg string, err error, status int) {
 	if err != nil {
-		log.Event(ctx, msg, log.Error(err), log.ERROR)
+		log.Event(ctx, err.Error(), log.Error(err), log.ERROR)
```

Comment:
if `err != nil` - the `msg` doesn't appear to go into the logs. I think there is some cases where it should be? If `log.Error(err)` prints the error string, I think maybe going back to `log.Event(ctx, msg, log.Error(err), log.ERROR)` may be better? (I may have just read it completely wrong 😅 )